frontend/src/static/js/v_authorship.js

```diff
@@ -95,6 +98,7 @@ window.vAuthorship = {
           out.segments = segments;
           res.push(out);
         }
+        this.totalLineCount = totalLineCount;
```

Comment:
i assume you are doing a `let totalLineCount` to avoid writing over `this.totalLineCount` multiple times.

if that is the case, then you should write it after everything, not after each file.
if that is not the case, i think `this.totalLineCount += lineCnt` is a cleaner option.

Reply:
i've changed the implementation to ensure ```this.totalLineCount``` is only updated after writing everything, and not after each file.